js/packages/ufg-client/src/resources/fetch-resource.ts

```diff
@@ -1,6 +1,6 @@
 import type {Cookie} from '../types'
 import {type Logger} from '@applitools/logger'
-import {makeReq, type Fetch, type Proxy, type Hooks} from '@applitools/req'
+import {makeReq, type Fetch, type Proxy, type Hooks, Response} from '@applitools/req'
```

Comment:
If you going to accept what I proposed you would have to remove it as well.
```suggestion
import {makeReq, type Fetch, type Proxy, type Hooks} from '@applitools/req'
```